docs/docs/system-administration/importing-and-exporting/organisations.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Data Migration
+title: Organisations
```

Comment:
Isn't there a way to set up redirects in docusaurus, just in case there are other links out there somewhere?

Reply:
I've done this [here](https://github.com/Flagsmith/flagsmith/pull/4177/commits/928667d7193bd6421e42ee2af506de86ccfd160d)

Reply:
the urls are based on file name I believe and not h1

Reply:
> I've done this [here](https://github.com/Flagsmith/flagsmith/pull/4177/commits/928667d7193bd6421e42ee2af506de86ccfd160d)

Ah ok thanks!

Reply:
You changed the file name too.